baybe/searchspace/continuous.py

```diff
@@ -109,6 +114,23 @@ def constraints_cardinality(self) -> tuple[ContinuousCardinalityConstraint, ...]
             if isinstance(c, ContinuousCardinalityConstraint)
         )
 
+    @property
+    def n_inactive_parameter_combinations(self) -> int:
+        """The number of possible inactive parameter combinations."""
+        return math.prod(
+            c.n_inactive_parameter_combinations for c in self.constraints_cardinality
+        )
+
+    def inactive_parameter_combinations(self) -> Iterable[frozenset[str]]:
+        """Iterate over all possible combinations of inactive parameters."""
+        for combination in product(
+            *[
+                con.inactive_parameter_combinations()
+                for con in self.constraints_cardinality
+            ]
+        ):
+            yield frozenset(chain(*combination))
```

Comment:
Analogous to many other cases where we return tuples instead of lists, i.e. immutable vs. mutable. I think for the return types of methods there is no clear choice what is better (unlike for the members itself, which we clearly prefer to be immutable). So I have no strong reason here, just following the convention used so far. Opinions?

Reply:
No strong opinion from me, was more out of curiousity, will hence resolve.